ios/SpectrumKitSample/Podfile.lock

```diff
@@ -1,29 +1,29 @@
 PODS:
   - libpng (1.6.18)
-  - libwebp (1.0.0):
-    - libwebp/core (= 1.0.0)
-    - libwebp/dec (= 1.0.0)
-    - libwebp/demux (= 1.0.0)
-    - libwebp/dsp (= 1.0.0)
-    - libwebp/enc (= 1.0.0)
-    - libwebp/mux (= 1.0.0)
-    - libwebp/utils (= 1.0.0)
-    - libwebp/webp (= 1.0.0)
-  - libwebp/core (1.0.0):
+  - libwebp (1.0.2):
+    - libwebp/core (= 1.0.2)
+    - libwebp/dec (= 1.0.2)
+    - libwebp/demux (= 1.0.2)
+    - libwebp/dsp (= 1.0.2)
+    - libwebp/enc (= 1.0.2)
+    - libwebp/mux (= 1.0.2)
+    - libwebp/utils (= 1.0.2)
+    - libwebp/webp (= 1.0.2)
+  - libwebp/core (1.0.2):
     - libwebp/webp
-  - libwebp/dec (1.0.0):
+  - libwebp/dec (1.0.2):
     - libwebp/core
-  - libwebp/demux (1.0.0):
+  - libwebp/demux (1.0.2):
     - libwebp/core
-  - libwebp/dsp (1.0.0):
+  - libwebp/dsp (1.0.2):
     - libwebp/core
-  - libwebp/enc (1.0.0):
+  - libwebp/enc (1.0.2):
     - libwebp/core
-  - libwebp/mux (1.0.0):
+  - libwebp/mux (1.0.2):
     - libwebp/core
-  - libwebp/utils (1.0.0):
+  - libwebp/utils (1.0.2):
     - libwebp/core
-  - libwebp/webp (1.0.0)
+  - libwebp/webp (1.0.2)
```

Comment:
There's a checksum at the end of this file that needs to be updated in addition to simply updating the version number. However it looks like the dependency we depend on [hasn't submitted a pod for v1.0.2 yet](https://cocoapods.org/pods/libwebp) - so we might need to hold this update (or at least the iOS part) for now until these changes are updated upstream.

Reply:
So we need some help from @bpoplauschi or @ShaharHD. :)

Reply:
I will look into it

Reply:
I just published the podspecs for 1.0.1 and 1.0.2 versions

Reply:
Awesome, thanks @bpoplauschi !

Reply:
Thanks!

@cuva coupd you generate checksums? Not at a machine at the moment